src/components/Editor.js

```diff
@@ -254,10 +286,23 @@ const Editor = React.createClass({
       click: () => this.showConditionalPanel(line)
     };
 
-    showMenu(e, [
+    let items = [
       toggleBreakpoint,
       conditionalBreakpoint
-    ]);
+    ];
+
+    if (bp) {
+      const disableBreakpoint = {
+        id: "node-menu-disable-breakpoint",
+        label: disableBpLabel,
+        accesskey: "D",
+        disabled: false,
+        click: () => this.toggleBreakpointDisabledStatus(line)
```

Comment:
I've been thinking about this a bit and came up with what I *think* is a nice improvement.

Lets create each context menu item with the specific action

e.g.

```js
const disableBreakpoint = {
         id: "node-menu-disable-breakpoint",
         label: disableBpLabel,
         accesskey: "D",
         disabled: false,
         click: () => this.props.disableBreakpoint({
              sourceId: ...,
              line: line + 1
         });
```

I think we might go this approach anyways because each item should have it's own ID as well. i.e. "node-menu-disable-breakpoint", and "node-menu-enable-breakpoint" https://github.com/devtools-html/debugger.html/issues/1228

We do the same thing already with `enableBreakpoint` so it's just a question of creating the other 8 or so items.

What do you think?